packages/components/addon/components/hds/pagination/compact/index.hbs

```diff
@@ -27,4 +27,13 @@
       @disabled={{@isDisabledNext}}
     />
   </nav>
+
+  {{#if this.showSizeSelector}}
+    <Hds::Pagination::SizeSelector
+      @pageSizes={{this.pageSizes}}
+      @label={{@sizeSelectorLabel}}
+      @selectedSize={{this.currentPageSize}}
```

Comment:
I may be missing something, but there doesn't seem to be a way to set `this.currentPageSize` (i.e. this component doesn't have an `@currentPageSize` param.

Reply:
@andrew-hashicorp you're right, I've missed it (for some reasons, I thought the sub-component was yielded). Let me investigate what needs to be done, I'll get back to you.

Reply:
I've opened a new PR to "fix" this oversight: 
https://github.com/hashicorp/design-system/pull/1724

(but I need to discuss with you a detail before sending it to review, I'll set up some time to discuss it today)